### DISCUSSION TITLE
EFI system partition confusion

### DISCUSSION THREAD

**Discussion by arky217** [SpiralLinux/SpiralLinux-project#480] [Q&A]
I have 2 distros already installed and I want to install Spiral on another partition.
When I try to install Spiral, I get the message that I must have a EFI system partition of type FAT32 with the 
boot flag set and at least 300MB in size.

Well, I already have the EFI system partition that the other 2 distros use and
it is type FAT32 and has the boot flag set, but the size is 256MB.

It looks like I can edit that EFI system partition and make it 300MB but I fear that since there is no
empty space between it and the first installed distro, that changing it from
256MB to 300MB will override part of that 1st distro and make in unbootable.

What should I do ?

**Comment by geckolinux** [comment]
Hi there, that's just a pointless warning that the installer shows. You can install it with your existing EFI partition.

**Comment by ghost** [comment]
And ... and and and The EFI Partition can be 100s of GigaBytes in size PLUS you can have multi EFIs per disk...   But please dont do that  : P    lol

I was playing around and created 2 EFIs  one to boot and a 1TB partition to hide my porn collection but some distros get confused by the non sense   : P   lol

CYa,
Bill
